src/pug/main/reviews-section.pug

```diff
@@ -0,0 +1,63 @@
+section.reviews-section
+  .container
+    .reviews-title-holder
+      .reviews-title-box
+        h2.title-2 Reviews
+  .reviews-slider-wrapper
+    .container
+      .rev-slider
+        .reviews-item
+          .rev-item-holder
+            .rev-img-wrapper
+              img.rev-img(src='./img/background-hero-2x.jpg' alt='')
```

Comment:
rename file.
add alt attribute.
add webp format
we need 1x image and 2x image as well